client/containers/Pub/PubDocument/Review/ReviewerDialog.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Button, Classes, Dialog, InputGroup } from '@blueprintjs/core';
+
+require('./reviewerDialog.scss');
+
+type OwnProps = {
+	isOpen: boolean;
+	pubData: {
+		title: string;
+	};
+	onClose: (...args: any[]) => any;
+	createReviewDoc: () => any;
+	setReviewTitle: any;
+	reviewTitle: string;
+	reviewerName: any;
+	setReviewerName: any;
+	canEdit: boolean;
+};
+
+const defaultProps = {};
+
+type Props = OwnProps & typeof defaultProps;
+
```

Comment:
What does `typeof defaultProps` do here, when defaultProps is {}? Not a rhetorical question, I really don't know what this does.

Reply:
me niether its a copy paste thing 🤷🏿‍♂️